comictalker/comictalkerapi.py

```diff
@@ -16,23 +16,32 @@
 from __future__ import annotations
 
 import logging
+import sys
 
-import comictalker.talkers.comicvine
-from comictalker.talkerbase import ComicTalker, TalkerError
-
-logger = logging.getLogger(__name__)
+if sys.version_info < (3, 10):
+    from importlib_metadata import entry_points
+else:
+    from importlib.metadata import entry_points
 
+from comictalker.talkerbase import ComicTalker
 
-def get_comic_talker(source_name: str) -> type[ComicTalker]:
-    """Retrieve the available sources modules"""
-    sources = get_talkers()
-    if source_name not in sources:
-        raise TalkerError(source=source_name, code=4, desc="The talker does not exist")
-
-    talker = sources[source_name]
-    return talker
+logger = logging.getLogger(__name__)
 
 
 def get_talkers() -> dict[str, type[ComicTalker]]:
-    """Returns all comic talker modules NOT objects"""
-    return {"comicvine": comictalker.talkers.comicvine.ComicVineTalker}
+    """Returns all comic talker plugins (internal and external)"""
+    talkers = {}
+
+    for ep in entry_points(group="comictagger_talkers"):
```

Comment:
This should be `comictagger.talkers` see https://packaging.python.org/en/latest/specifications/entry-points/#data-model